maintenance_page/manifests/production/ingress_external_to_main.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: apply-for-qts-in-england.education.gov.uk
```

Comment:
afqts doesn't have any external ingresses, so they can all be removed i.e. null_host_header = false